default_config.toml

```diff
@@ -0,0 +1,31 @@
+[main]
```

Comment:
This file was updated in #610 to ensure that no configuration options are being set silently. It was also renamed to `example_config.toml`.

Reply:
Sorry @antalszava , I made this PR in a big hurry and should not have done so! I hope it is all fine now!

Reply:
No worries!